lib/active_admin/application.rb

```diff
@@ -271,10 +271,12 @@ def controllers_for_filters
   private
 
     def register_default_assets
-      register_stylesheet 'active_admin.css',       media: 'screen'
-      register_stylesheet 'active_admin/print.css', media: 'print'
+      ActiveSupport::Deprecation.silence do
```

Comment:
Although after a second though, the current approach looks fine too! Do what you think it's best...